MusicHub/users/profile_service.py

```diff
@@ -1,5 +1,7 @@
 from rest_framework.serializers import ValidationError
 
+MAX_PICTURE_SIZE = 3000000  # value in bytes, max 3Mb
```

Comment:
When you write numbers with more than 3 digits by hand you can use underscore `_` as separator for better readability:
```
>>> 12345678 == 12_345_678
True
```